todo_app/app.py

```diff
@@ -19,10 +22,18 @@ def create_app():
     client_secret = os.getenv("Git_Client_secret")
     client_id = os.getenv('Git_Client_ID')
     client = WebApplicationClient(client_id)
+    
+    app.logger.setLevel(app.config['LOG_LEVEL'])
+    app.logger.info("Value of logger is info")
```

Comment:
Your log message will sometimes be a lie. It will end up appearing even if LOG_LEVEL is DEBUG.
```suggestion
    app.logger.info(f"Value of logger is {app.config['LOG_LEVEL']}")
```